interface/ceed-jit-tools.c

```diff
@@ -287,7 +287,9 @@ int CeedGetJitAbsolutePath(Ceed ceed, const char *relative_file_path, char **abs
     CeedCall(CeedCheckFilePath(ceed, *absolute_file_path, &is_valid));
 
     if (is_valid) return CEED_ERROR_SUCCESS;
+    // LCOV_EXCL_START
     else CeedCall(CeedFree(absolute_file_path));
```

Comment:
Not sure I understand why we'd want to free the array, but only when there's an error.

Reply:
It isn't freeing when there is an error exactly. `CeedCheckFilePath` is seeing if we generated a real filepath from the given relative path and the current root dir we are checking it with. If the generated absolute path isn't valid, then we need to free the generate (bad) absolute path and make a new one with the next root dir